lib/v2/scitation/journal.js

```diff
@@ -0,0 +1,69 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const config = require('@/config').value;
+const path = require('path');
+const { art } = require('@/utils/render');
+
+const { CookieJar } = require('tough-cookie');
+const cookieJar = new CookieJar();
+
+module.exports = async (ctx) => {
+    const pub = ctx.params.pub;
+    const jrn = ctx.params.jrn;
+    const host = 'https://'.concat(pub, '.scitation.org');
+    const jrnlUrl = host.concat('/toc/', jrn, '/current?size=all');
+
+    const response = await got(jrnlUrl, {
+        cookieJar,
+    });
+    const $ = cheerio.load(response.data);
+    const jrnlName = $('.header-journal-title').text();
+    const list = $('.card')
+        .map((_, item) => {
+            $(item).find('.access-text').remove();
+            const title = $(item).find('.hlFld-Title').text();
+            const authors = $(item).find('.entryAuthor.all').text();
+            const img = $(item).find('img').attr('src');
+            const link = $(item).find('.ref.nowrap').attr('href');
+            const doi = link.replace('/doi/full/', '');
+            return {
+                title,
+                link,
+                doi,
+                authors,
+                img,
+            };
+        })
+        .get();
+
+    const renderDesc = (item) =>
+        art(path.join(__dirname, 'templates/description.art'), {
+            item,
+        });
+    await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const response2 = await got.post(host.concat('/action/PB2showAjaxAbstract'), {
+                    cookieJar,
+                    body: 'doi=' + item.doi,
+                    headers: {
+                        'X-Requested-With': 'XMLHttpRequest',
+                        'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8',
+                        'Accept-Encoding': 'gzip, deflate, br',
+                        'User-Agent': config.ua,
```

Comment:
```suggestion
```
No need to specify UA again
https://github.com/DIYgod/RSSHub/blob/213b1140f03c75336f9f0fdf9d3150209db97863/lib/utils/got.js#L34